ReservationEngine/Dockerfile.develop

```diff
@@ -10,4 +10,6 @@ RUN dotnet restore "app.csproj"
 COPY . .
 RUN dotnet build --no-restore -c $BUILD_CONFIGURATION
 
-ENTRYPOINT ["dotnet", "out/app.dll"]
\ No newline at end of file
+RUN echo "exec dotnet run --no-build --no-launch-profile -c $BUILD_CONFIGURATION -- \"\$@\"" > /entrypoint.sh
```

Comment:
>echo [](start = 4, length = 4)

This echo statement is in our default Dockerfile.develop template?

Reply:
Yep, using an intermediate shell script seems to be needed to use Docker's variable substitution. More details in this PR: [https://devdiv.visualstudio.com/DevDiv/_git/Mindaro/pullrequest/134084?_a=overview](https://devdiv.visualstudio.com/DevDiv/_git/Mindaro/pullrequest/134084?_a=overview)

---
In reply to: [266646275](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/20#discussion_r266646275) [](ancestors = 266646275)